StockpileStatisticsHooks.cs

```diff
@@ -56,21 +56,21 @@ static void Postfix(Stockpile __instance, bool __result, ushort type, int amount
 		}
 	}
 
-	[HarmonyPatch(typeof(Stockpile))]
-	[HarmonyPatch("TryRemoveFood")]
-	public class StockpileHookTryRemoveFood
-	{
+	[HarmonyPatch(typeof(ColonyShopVisitTracker))]
+	[HarmonyPatch("OnVisit")]
+	public class ColonyShopVisitTrackerHookOnVisit
+    {
 		public  static Stockpile inFunctionStockpile = null;
 		public static bool isFood;
 
-		static void Prefix(Stockpile __instance, ref float currentFood, float desiredFoodAddition)
+		static void Prefix(ref NPCBase npc, ref bool gotFood)

```

Comment:
`npc` shouldn't be `ref` here, remove it.
gotFood.. I think we can make it `out`?

Reply:
I wasn't able to change gotFood to an out, it gave a compiler error as we aren't setting that value in our method. 

Would it be better to leave it as a ref, remove it, or is there a better way to get it as a out?